README.md

```diff
@@ -61,31 +61,37 @@ FQM-Testify is a React-based web application for analyzing FHIR-based electronic
+
+(1) Resources can be created by selecting a resource from the dropdown list of resource templates. These resource templates are based on the FHIR `dataRequirements` of the uploaded measure (`dataRequirements` calculations and all other FHIR eCQM calculation in the app is done using [the fqm-execution library](https://github.com/projecttacoma/fqm-execution)). Upon selection, a code editor modal will open, pre-populated with the resource. The user can edit the resource and save it for future use.
+
+(2) Resources can be created by clicking on the "Add New Custom Resource Button." This option allows the user to paste a FHIR Resource into a code editor modal and make necessary edits before saving it as a resource for the patient.
+
+A FHIR resource can be examined, edited, and deleted by selecting the patient the resource belongs to, and then accessing the FHIR resource in the middle panel.
 
 ### Exporting a Test Case
 
-FQM-Testify allows for exporting of all test cases by clicking the "Download All Patients" button, which generates and downloads a zip file with a subdirectory for each Patient, containing a Patient Bundle including the patient and all resources which reference the patient. Individual Patient Bundles can be downloaded as well by clicking on the patient entry and selecting the download button.
+FQM-Testify allows for exporting of all test cases by clicking the "Download All" button in the left panel, which generates and downloads a `.zip` file with a subdirectory for each patient, containing a Patient Bundle including the patient and all resources which reference the patient. Individual Patient Bundles can be downloaded as well by clicking on the patient entry and selecting the download button.
 
 ### Running Measure Calculation on a Test Case
 
-FQM-Testify allows for running measure calculation on a single test case for a given measure by selecting a patient entry and clicking the calculate button. Clicking the calculate button will display logic highlighting using [the fqm-execution library](https://github.com/projecttacoma/fqm-execution). The measure calculation can be regenerated if non-patient test resources are added by clicking the calculate button. The logic highlighting can be hidden and shown using the hide/show logic highlighting button next to the calculate button.
+FQM-Testify allows for running measure calculation on a single test case for a given measure. Selecting a patient entry triggers calculation and displays logic highlighting using [the fqm-execution library](https://github.com/projecttacoma/fqm-execution) in the right panel. The measure calculation automatically regenerates if non-patient test resources are added to the patient.
```

Comment:
You can also add in that it automatically generates when the patient is edited or when a test resource is edited! Also if the measure bundle and measurement period are changed.